wasm2c/examples/fac/fac.c

```diff
@@ -468,6 +551,20 @@ static void init_tags(void) {
 
 static u32 w2c_fac(Z_fac_instance_t*, u32);
 
+static void init_globals(Z_fac_instance_t* instance) {
+}
+static void init_memory(Z_fac_instance_t* instance) {
+}
+
+static void init_data_instances(Z_fac_instance_t *instance) {
+}
+
+static void init_table(Z_fac_instance_t* instance) {
+}
+
+static void init_elem_instances(Z_fac_instance_t *instance) {
+}
```

Comment:
It it worth eliding all these function when they are empty (to make the output more readable?)

Reply:
Sounds good, done in #1999 which hopefully we can do as a follow-on to this one.